wrappers/src/schema_cs.cpp

```diff
@@ -62,7 +62,7 @@ REALM_EXPORT Schema* schema_create(SchemaObject* objects, int objects_length, Sc
                 Property p;
                 p.name = property.name;
                 p.type = property.type;
-                p.object_type = property.object_type ?: "";
+                p.object_type = property.object_type ? property.object_type : "";
```

Comment:
Apparently MSVC doesn't like the other syntax. Functionally it's equivalent, though.